index.html

```diff
@@ -0,0 +1,150 @@
+              <h3 class="program-title">Personal Training</h3>
+              <p class="program-detail">Members of the professional personal training staff at SuperPower Fitness
+                Centers are highly trained and
+                experienced in the field of exercise.</p>
+            </div>
+            <div class="program-tag">
+              <i class="program-img fa-solid fa-tent card-icon"></i>
+              <h3 class="program-title">Boot Camp</h3>
+              <p class="program-detail">For those seeking an intense, no-excuses workout that will take your fitness to
+                a new level.</p>
+            </div>
+            <div class="program-tag">
+              <i class="program-img fa-solid fa-dumbbell card-icon"></i>
+              <h3 class="program-title">Equipment</h3>
+              <p class="program-detail">We offer a large selection of cardiovascular, strength and free weight
+                equipment. You are sure to find some
+                of your favorites and try out new options to jazz up your workout.</p>
+            </div>
+            <div class="program-tag">
+              <i class="program-img fa-solid fa-person-rays card-icon"></i>
+              <h3 class="program-title">Exercise Classes</h3>
+              <p class="program-detail">Enjoy fun and challenging group fitness classes led by certified motivating
+                instructors</p>
+            </div>
+            <div class="program-tag">
+              <i class="program-img fa-solid fa-user-nurse card-icon"></i>
+              <h3 class="program-title">Health Assessment & Prescription</h3>
+              <p class="program-detail">Meet one on one with a degreed exercise professional to assess your fitness
+                status and set a personalized
+                exercise plan based on your goals</p>
+            </div>
+          </div>
+          <button type="button" class="join-btn">Join SPFC 2023 Now</button>
+          <a href="#" class="desktop-link max-font-26">see the whole program</a>
+        </div>
+      </div>
+    </section>
+
+    <section id="trainers" class="flex column">
+      <div class="wrapper flex column">
+        <h2 class="title max-font-26">Featured Trainers</h2>
+        <div id="trainers-container" class="trainers-container flex column">
+
+        </div>
+        <button type="button" class="trainer-btn"><span class="moreless">MORE </span><i
+            class="chevron fas fa-chevron-down"></i></button>
+      </div>
+    </section>
+
+    <section id="partners">
+      <div class="container">
+        <h2 class="title max-font-26">Partner</h2>
+        <div class="company flex">
+          <span id="facebook">Facebook</span>
+          <span id="mozilla">Mozilla</span>
+          <span id="apple">Apple</span>
+          <span id="microsoft">Microsoft</span>
+          <span id="google">Google</span>
+        </div>
+      </div>
+    </section>
```

Comment:
### Your Partner's section
![image](https://user-images.githubusercontent.com/93039359/235135544-f574e5ab-ef4f-421e-8ba5-4393ad70fb20.png)

- [ ] I guess while fixing the texts you have removed the image :fearful: Kindly add images here that fit the size of this box or add some professional pictures of trainers here to make your project look like the given design :+1: